docs/minio-admin-complete-guide.md

```diff
@@ -10,6 +10,7 @@ policy       manage canned policies
 config       manage configuration file
 heal         heal disks, buckets and objects on MinIO server
 top          provide top like statistics for MinIO
+prometheus   provide prometheus jwt token
```

Comment:
This is an incorrect description, we should say that `mc admin prometheus` is to manage prometheus metrics on server

Reply:
This jwt is only a sub command we are going to have more changes here. Such as bring this as a way to enable disable metrics etc. Change the prometheua auth style on server etc.

Reply:
Done :+1: